oneview/resource_server_profile.go

```diff
@@ -375,25 +387,93 @@ func resourceServerProfile() *schema.Resource {
 							Type:     schema.TypeString,
 							Optional: true,
 						},
-						"volume_shareable": {
-							Type:     schema.TypeBool,
-							Optional: true,
-						},
-						"volume_description": {
-							Type:     schema.TypeString,
-							Optional: true,
-						},
-						"volume_provision_type": {
+						"state": {
 							Type:     schema.TypeString,
 							Optional: true,
 						},
-						"volume_provisioned_capacity_bytes": {
+						"status": {
 							Type:     schema.TypeString,
 							Optional: true,
 						},
-						"volume_name": {
-							Type:     schema.TypeString,
+						"volume": {
+							Type:     schema.TypeSet,
 							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"initial_scope_uris": {
+										Type:     schema.TypeString,
+										Optional: true,
+									},
+									"is_permanent": {
+										Type:     schema.TypeBool,
+										Optional: true,
+									},
+									"templateUri": {
+										Type:     schema.TypeString,
+										Optional: true,
+									},
+									"properties": {
+										Type:     schema.TypeSet,
+										Optional: true,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"volume_set": {
```

Comment:
Write the struct attributes alphabetically . Volumeset should come below. Also many attributes are missing. Ispinned, Iscompressed. Please check the golang struct to get the list of full attributes

Reply:
done

Reply:
Schema is not alphabetically arranged.
and SP Schema is updated.